index.bs

```diff
@@ -779,6 +782,16 @@ spec:dom-ls; type:interface; text:Document
     dangerous and should not be used if you have Real Deal auth and csrf cookies
     used.
 
+   * '<dfn>`unsafe-include-credentials`</dfn>' All cross-origin requests will
```

Comment:
Are we sure this is what you want? I would have thought this would be 


All cross-origin requests to the parent physical origin will automatically have their credentials mode set to include.

Additionally, what happens if JS sets it to false? We should clarify that here (even if it is to say, this will be ignored.)